lib/rules/no-identical-title.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var utils = require("../utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function isFirstArgLiteral(node) {
+    return node.arguments && node.arguments[0] && node.arguments[0].type === "Literal";
+}
+
+
+function handlTestCaseTitles(context, titles, node, title) {
+    if (utils.isTest(node.callee)) {
+        if (titles.indexOf(title) !== -1) {
+            context.report({
+                node: node.arguments[0],
+                message: "Test title is used multiple times in the same test module."
```

Comment:
Nitpick, but I'd love to see this say "Test name" instead of "Test title", as well as just "module" instead of "test module".